doc/source/download.rst

```diff
@@ -256,6 +256,23 @@ available at https://anaconda.org/conda-forge/gdal.
 
     conda install -c conda-forge gdal
 
+Vcpkg
+................................................................................
+
+The gdal port in vcpkg is kept up to date by Microsoft team members and community contributors.
+The url of vcpkg is: https://github.com/Microsoft/vcpkg .
+You can download and install gdal using the vcpkg dependency manager:
+
+::
+
+    git clone https://github.com/Microsoft/vcpkg.git
+    cd vcpkg
+    ./bootstrap-vcpkg.sh  # ./bootstrap-vcpkg.bat for Windows
+    ./vcpkg integrate install
```

Comment:
AFAICT this command has no effect outside Window/MSVS. What users really need is using the CMake toolchain file provided by vcpkg.
FTR vcpkg will build gdal and all dependencies from source, not just install binary packages.